docs/examples/model_repository/resnet_torchscript/Readme.md

```diff
@@ -0,0 +1,154 @@
+# Loading Pytorch Models To Nvidia Triton Inference Server
+
+Pytorch models are supported by the Nvidia [Triton Inference Server](https://github.com/triton-inference-server/). To Load the Pytorch model to triton inference server one must convert the model either ONNX format or in the TorchScript format.
```

Comment:
Is it accurate that the PyTorch model must be converted with either ONNX or TorchScript to be usable by Triton? Can a plain Python model be used?

Reply:
Starting in 20.10 we also have a python backend that can run arbitrary python code. It is not performance focused at this point but does allow you to use pytorch directly. See https://github.com/triton-inference-server/python_backend.
If you are feeling adventurous you can build it yourself and try it with 20.09.

Reply:
Thank you it's good to know, I'll give it a shot.

Reply:
@deadeyegoodwin  Here I am savin the model post-conversion to Torchscript.  So this can be loaded by the Triton server

Reply:
We are in the middle of a release so I haven't looked at it yet. We are also updating the docs and the getting started content so it may be necessary to adjust it some based on that. But I hope to be able to look at it within the next week or two.